components/fieldgroups/profile/marketo-engage-person-details.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/marketo-engage-person-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Engage Person Details",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "This field group is used to capture Marketo Engage cookie IDs for the B2B Persons.",
+  "definitions": {
+    "marketo-engage-person-details": {
+      "properties": {
+        "xdm:marketoEngage": {
```

Comment:
Can you move this under `"xdm:b2b"` category?

Reply:
Like this?
```
  "definitions": {
    "marketo-engage-person-details": {
      "properties": {
        "xdm:b2b": {
          "type": "object",
          "properties": {
            "xdm:marketoEngage": {
              "type": "object",
              "properties": {
                "xdm:cookieIDs": {
                  "title": "Cookie IDs",
                  "description": "Comma separated Marketo Engage cookie IDs.",
                  "type": "string"
                }
              }
            }
          }
        }
      }
    }
  }
```

Reply:
@anandphatak will it be an issue that the `XDM Business Person Details` field group already declares `xdm:b2b` at the same level? https://github.com/adobe/xdm/blob/master/components/fieldgroups/profile/b2b-person-details.schema.json#L22

We are unsure if this will cause a conflict, or if the underlying objects/fields would simply be merged together.

Reply:
I implemented the change, please let me know if it looks good to you.

Reply:
Yes they will get merged.